qa/tasks/ceph_manager.py

```diff
@@ -1965,7 +1965,7 @@ def get_pool(self):
         """
         with self.lock:
             if self.pools:
-                return random.choice(self.pools.keys())
+                return random.sample(self.pools.keys(), 1)[0]
```

Comment:
shouldn't you use `iter` here to access the first element?

Reply:
no need to do so, both Python2 and Python3 return a list, so we are safe here.

Reply:
```
>>> print(random.sample.__doc__)
Chooses k unique random elements from a population sequence.

        Returns a new list containing elements from the population while
        leaving the original population unchanged.  The resulting list is
        in selection order so that all sub-slices will also be valid random
        samples.  This allows raffle winners (the sample) to be partitioned
        into grand prize and second place winners (the subslices).
```